main.go

```diff
@@ -309,9 +328,14 @@ func main() {
 			fmt.Printf("couldn't collect metrics: %v", err)
 			os.Exit(1)
 		}
-		p.print(metrics)
+		err = p.print(metrics)
+		if err != nil {
+			fmt.Printf("couldn't print metrics: %v", err)
+			os.Exit(1)
+		}
 		if time.Now().After(until) {
 			break
 		}
 	}
+	p.close()
```

Comment:
This will only be executed if line 337 is reached, in all other cases it won't be called (error occurs or SIGINT is caught). Not the end of the world though, the function doesn't do anything anyway 😄

If we print to stdout we'll have no worries with closing the file and the caller can divert the output, I recommend doing this at some point.